src/Tribe/Utils/Post_Collection.php

```diff
@@ -24,6 +36,216 @@ public function __construct( array $items ) {
 		parent::__construct( array_filter( array_map( 'get_post', $items ) ) );
 	}
 
+	/**
+	 * Plucks fields from the posts in the collection creating a map using a field value as key and one
+	 * or more fields as values.
+	 *
+	 * Note: the method does not make any check on the uniqueness of the fields used as keys, e.g. this will
+	 * probably not return what intended: `$collection->pluck_combine( 'post_status', 'post_title' );`.
+	 * If there's a chance of the key fields not being unique, then use `#` as key field to simply return an
+	 * array of plucked values.
+	 *
+	 * @since TBD
+	 *
+	 * @param string                            $key_field    The field to key the return map by, or `#` to use
+	 *                                                        progressive integers to key the return value. Use fields
+	 *                                                        as keys only when their uniqueness is sure.
+	 * @param string|array<string>|array<array> $value_fields Either a single field name to populate the values with;
+	 *                                                        a list of fields, each plucked with default settings;
+	 *                                                        a map of fields to fetch, each defining a `single` and
+	 *                                                        `args` key to define the pluck `$single` and `$args`
+	 *                                                        parameters where applicable.
+	 *                                                        Additionally an `as` parameter can be specifed to alias
+	 *                                                        the field in the results.
+	 *
+	 * @return array<int|string,string|array> A list of plucked fields or a map of plucked fields keyed by the
+	 *                                        specified
```

Comment:
The comments look fine when directly looking at the file, but here specified field. could be on one line.